source/docs/casper/resources/tutorials/advanced/cross-contract.md

```diff
@@ -0,0 +1,538 @@
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# Cross Contract Communication
+
+Level of the tutorial: `Advanced`
+
+Time to complete the tutorial: `45 - 60 Minutes`
+
+Important topics to read before going further:
+
+- [getting started](../../../developers/writing-onchain-code/getting-started.md)
+- [writing on-chain code](../../../developers/writing-onchain-code/simple-contract.md)
+- [calling contracts](../../../developers/writing-onchain-code/calling-contracts.md)
+
+This will assure that you have needed tools and base knowledge to complete this tutorial. If not everything was clear while reading these sections, don't worry as we will be doing a full walkthrough step by step.
```

Comment:
Make tings clear. Explicitly state the infrastructure and tools. We should *not* be pointing people to three tutorials in order to  make sure they have the tools installed. 
Example
Prerequisites: 
Rust, CMake - link to installing these
Casper crate - link to installing this 
Signer , funded account - link to doing this

Reply:
Links will be added to the appropriate prerequisites after it gets merged. For now, one sentence has been introduced, to show where the link will find it's place